### PR TITLE
Ft admin auth ui 160556087

### PR DESCRIPTION
**Pull request by davidshare**
#### What does this PR do?
Create the user interface for creating new Admin and Admin signin

#### Description of Task to be completed?
- create an HTML file to host the new admin form
- add form fields for creating a new admin
- adjust stying for select form element in forms.css
- create an HTML file to host the admin signin form
- add form fields for admin to signin
- update links in main nav for admin pages

#### How should this be manually tested?
clone the repo and navigate to the root directory using the command line
navigate to the UI directory in the project directory
navigate to the Admin directory
open signin.html and new_admin.html file using a web browser

#### What are the relevant pivotal tracker stories?
[#160556087](insert url to user story)
[#160556121](insert url to user story)

### CODE REVIEW

File: UI/admin/signin.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </p>
+      </form>
+      <p class="forgot-pass-link"><a href="password_recover.html"> Forgot password</a></p>
+      <p>No account yet? <a href="../signup.html">Signup for free</a></p>
+    </div>
+  </section>
+  <script src="../js/main.js"></script>
+</body>
+
+</html>
```
Comment (KaiserPhemi) [inline UI/admin/signin.html]: #### Nitpick
New line after last line

File: UI/admin/new_admin.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <p class="form-control">
+          <button class="btn btn-danger" id="signup-btn">Signup</button>
+        </p>
+      </form>
+    </div>
+  </section>
+  <script src="../js/main.js"></script>
+</body>
+
+</html>
```
Comment (KaiserPhemi) [inline UI/admin/new_admin.html]: #### Nitpick
New line after last line

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
Implement Feedback

File: UI/admin/new_admin.html (hunk 2 above)
Comment (davidshare) [reply UI/admin/new_admin.html]: Feeback implemented

File: UI/admin/signin.html (hunk 1 above)
Comment (davidshare) [reply UI/admin/signin.html]: Feeback implemented

**Review by KaiserPhemi — APPROVED** [review]

**Comment by KaiserPhemi** [comment]
# LGTM
👍

**Pull request merged**